lib/security/JweToken.ts

```diff
@@ -97,6 +219,25 @@ export default class JweToken extends JoseToken {
     return encrypt(keyBuffer, jwk);
   }
 
+  /**
+   * Gets the header as a JS object.
+   */
+  public getHeader (): any {
+    if (this.isFlattenedJSONSerialized) {
+      let headers = this.unprotected;
+      if (!headers) {
+        headers = {};
+      }
+      if (this.protected) {
+        const jsonString = Base64Url.decode(this.protected);
+        const protect = JSON.parse(jsonString) as {[key: string]: any};
```

Comment:
Nit: rename to `protectedHeaders`, also, is is necessary to `as {[key: string]: any}`? Would `JSON.parse(jsonString)` suffice?

Reply:
JSON.parse returns any as it can also parse direct strings, booleans, and numbers to those types.